docs/features/sharepoint.md

```diff
@@ -0,0 +1,81 @@
+# Sharepoint File Ingestion
+
+## Overview
+
+A new logic app has been created to ingest sharepoint documents into a blob container for Information Assistant to process. It starts with the folder provided and crawls down through each folder in the sharepoint site. This process runs once every 24 hours.
+
+### Restrictions
+
+#### No RBAC
+In order to access your files in Sharepoint, you will have to login via Entra with the user that has access to these sharepoint files. We recommend creating a new user to do this.
+
+#### Document Support
+This ingestion process is only for **documents supported by our current pipeline**, the current officially supported file types are; pdf, docx, html, htm, csv, md, pptx, txt, json, xlsx, xml, eml, msg. This ingestion process does **NOT** include lists or information found on pages in your sharepoint site. It is only for documents.
+
+#### Files are Stored in Blob Storage
+Your sharepoint files will be stored in the blob storage created by Information Assistant, they are not going directly into AI Search. **There will be a copy of all your sharepoint files you are ingesting in blob storage.**
+
+### New Local Dev Variables
+
+#### Sharepoint Sites
+
+This is a comma delimited list of sharepoint site(s) you wish to ingest. **It shares a one-to-one relationship with the sharepoint folders variable.** If you have one sharepoint site, you need one sharepoint folder and vice versa. You may list the same site twice if you wish to more granular control over what is being crawled in sharepoint.
+
+#### Sharepoint Folders
+
+This is a comma delimited list of sharepoint folder(s) you wish to be your entry point for the ingestion. **It shares a one-to-one relationship with the sharepoint sites variable.** If you have one sharepoint folder, you need one sharepoint site and vice versa. It will crawl this folder and then automatically crawl any folders beneath it. The sharepoint site root is typically /Shared Documents
+
+#### Accepted File Types
+
+The sharepoint ingestion uses the same ingestion pipeline as Information Assistant to process the files obtained from sharepoint. The default values provided are those that have official support; pdf, docx, html, htm, csv, md, pptx, txt, json, xlsx, xml, eml, msg
+
+## Usage Instructions
+
```

Comment:
Wasn't aware this had manual deployment steps.. this will need to be added to the "Deployment" markdown as well.